### PR TITLE
[CalendarService] Add documentation

### PR DESCRIPTION
**Pull request by VasiliyPetrov1**
Add documentation to all CalendarService methods
besides getExpiredEvents()

### CODE REVIEW

File: src/main/java/ua/kpi/iasa/IASA_Organiser/service/CalendarService.java [hunk 1]
```diff
@@ -2,20 +2,32 @@
...
 
+
+/**
+ * This class is used for making basic operations on events {@link Event}
+ * when we need to consider their state, defined by the state of their fields
+ * like {@code date}, {@code time} and {@code priority} {@link Priority} etc.
+ * For example, we can find special events, sort Events by given feature
+ * and get the sequence of all events with special characteristics given
+ * by the user via methods declared in this class.
+ */
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/service/CalendarService.java:30]: Add @author and @see

File: src/main/java/ua/kpi/iasa/IASA_Organiser/service/CalendarService.java [hunk 2]
```diff
@@ -2,20 +2,32 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import ua.kpi.iasa.IASA_Organiser.controller.Controller;
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/service/CalendarService.java:5]: Redundant import

File: src/main/java/ua/kpi/iasa/IASA_Organiser/service/CalendarService.java [hunk 3]
```diff
@@ -2,20 +2,32 @@
...
 import ua.kpi.iasa.IASA_Organiser.data.GenericDataManager;
 import ua.kpi.iasa.IASA_Organiser.data.impl.DefaultFileDataManager;
 import ua.kpi.iasa.IASA_Organiser.model.Calendar;
 import ua.kpi.iasa.IASA_Organiser.model.Event;
 import ua.kpi.iasa.IASA_Organiser.model.Priority;
 import ua.kpi.iasa.IASA_Organiser.model.Tag;
 import ua.kpi.iasa.IASA_Organiser.model.Type;
+import ua.kpi.iasa.IASA_Organiser.view.View;
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/service/CalendarService.java:13]: Redundant import

File: src/main/java/ua/kpi/iasa/IASA_Organiser/service/CalendarService.java [hunk 4]
```diff
@@ -2,20 +2,32 @@
...
 
 import java.time.LocalDate;
 import java.time.LocalTime;
 import java.util.Comparator;
+import java.util.GregorianCalendar;
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/service/CalendarService.java:18]: Redundant import

**Review by MLGMAG — CHANGES_REQUESTED** [review]

**Review by MLGMAG — APPROVED** [review]

**Review by makrushinoff — APPROVED** [review]

**Pull request merged**